.env_sample

```diff
@@ -0,0 +1,18 @@
+GOOGLE_CLIENT_ID= please enter your google client id here
+GOOGLE_CLIENT_SECRET=Rplease enter your google client secret here
+GOOGLE_CALLBACK_URL=please enter your google callback url here
+FACEBOOK_CLIENT_ID=please enter your facebook client id here
+FACEBOOK_CLIENT_SECRET=please enter your facebook client secret here
+FACEBOOK_CALLBACK_URL=please enter your facebook callback url here
+SESSION_SECRET=please enter your session secret keyword here
+JWT_KEY=please enter your jwt secret keyword here
+TOKEN_SECRET=please enter your jwt secret keyword here
+TWITTER_CUSTOMER_KEY=please enter your twitter TWITTER_CUSTOMER_KEY here
+TWITTER_CUSTOMER_SECRET=please enter your TWITTER_CUSTOMER_SECRET here
+TWITTER_CALLBACK_URL=please enter your TWITTER_CALLBACK_URL here
+SENDGRID_API_KEY=please enter your SENDGRID_API_KEY here
+
+PROD_DB_URL=please enter your Production database url here
+EMAIL_SECRET=please enter your EMAIL_SECRET here
+DEV_DB_URL=please enter your development database url here
+TEST_DB_URL=please enter your TEST database url here
```

Comment:
Nice work on creating the .env.sample file. But please, endeavour to enter a new line at the end of line 18.

Reply:
Thank you @pitaz for the feedback feedback would be implemented as soon as possible

Reply:
Nice work @edogbosunny, on line 14 endeavour to close it up